### ISSUE TITLE
Refactor CSS to use PostCSS instead of SASS

### ISSUE THREAD

**Issue by taras** [offirgolan/ember-light-table#103]
PostCSS will allow us to use their [autoprefixer](https://github.com/postcss/autoprefixer) module which would automatically add prefixed to flex box. It would address #97.

Objections?

**Comment by offirgolan** [comment]
That sounds good to me. Can you handle this task @taras?

**Comment by taras** [comment]
@offirgolan yep, I'll take care of it. I just wanted to check with you that you're good with this.

**Comment by offirgolan** [comment]
Oh okay gotcha. Yeah this sounds good. Prolly should have been done before 1.0 but it never even crossed my mind

**Comment by offirgolan** [comment]
@taras I made a temporary fix in #163 by just autoprefixing the addon.css
